spring-integration-smb/build.gradle

```diff
@@ -56,7 +56,7 @@ compileTestJava {
 ext {
 	idPrefix = 'smb'
 
-	jcifsVersion = '1.3.18.3'
+	jcifsVersion = '2.1.7'
 	log4jVersion = '2.11.0'
 	springIntegrationVersion = '5.0.13.BUILD-SNAPSHOT'
```

Comment:
So you need this version of SMB to work with this old Spring Integration version?

If not, we should bump this to 5.1.4.RELEASE.

Reply:
OK with me, but let's address that in the separate issue!
Seems for me there is some incompatibility.

Reply:
Ok; makes sense; but we **will** need to bump `gradle.properties` to 1.1.0.BUILD-SNAPSHOT for this dependency change.

Reply:
Yep! And I guess we even can release this extension afterward to meet community requirements and make their lives easier with SMB 2/3 support 😄

Reply:
We are currently using the 5.0.3.RELEASE right now. Is there much difference between this version and the one you are proposing?

Reply:
>We are currently using the 5.0.3.RELEASE 

@GregBragg you should really try to keep more up-to-date with releases, at least within a minor release (5.0.x). 5.0.3 is over a year old 5.0.13 represents 10 bug-fix releases since then (mostly minor, but some are important).

While I would prefer bumping this to 5.1.x, I don't have a major objection to releasing this against 5.0.13 and quickly following up with 1.2.0.RELEASE against 5.1.4 and 1.3.0.BUILD-SNAPSHOT against 5.2.x (which will get its first milestone soon).

(Thanks for all your work on this, BTW).